lib/enkf/local_dataset.cpp

```diff
@@ -135,3 +135,36 @@ int local_dataset_get_size( const local_dataset_type * dataset ) {
 hash_iter_type * local_dataset_alloc_iter(const local_dataset_type * dataset) {
   return hash_iter_alloc(dataset->nodes);
 }
+
+bool local_dataset_has_row_scaling(const local_dataset_type * dataset, const char * key) {
+  return false;
+}
+
+stringlist_type * local_dataset_alloc_scaled_keys(const local_dataset_type * dataset) {
```

Comment:
should return reference of type `std::vector<std::string> &`

Reply:
Not a reference I guess :-)